content/en/apps/concepts/interoperability.md

```diff
@@ -3,36 +3,52 @@ title: Interoperability
 linkTitle: "Interoperability"
 weight: 8
 description: >
-  Exchanging information between CHT Core and other systems
+  Two way data synchronization between the CHT Core and other health systems. 
 keywords: interoperability integrations fhir icd openhie openhim
 relatedContent: >
   apps/features/integrations
 ---
 
-# Introduction
+## Introduction 
 
-Interoperability is the ability of health information systems to work together, even if they weren't specifically designed to work together. With interoperability, patient information can be seen, exchanged, and used across different platforms. This is different from _integration_ which requires custom development to connect two specific systems together. 
+Interoperability refers to the ability of different systems and applications to communicate with each other and exchange data seamlessly.With interoperability, patient information can be seen, exchanged, and used across different platforms that may be disparate or relatively similar. The information/data exchanged has to be understood across the different software for these systems to become interoperable. 
```

Comment:
```suggestion
Interoperability refers to the ability of different health information systems and applications to communicate with each other and exchange data seamlessly. With interoperability, patient information can be seen, exchanged, and used across different platforms. The information/data exchanged has to be understood across the different software for these systems to become interoperable. This is different from _integration_ which requires custom development to connect two specific systems together.
```

Putting back some text from the previous version on integration, as it seems important to mention here.